package.json

```diff
@@ -2,13 +2,15 @@
   "name": "nix-installer-action",
   "version": "1.0.0",
   "description": "You can use [`nix-installer`](https://github.com/DeterminateSystems/nix-installer) as a Github action:",
-  "main": "lib/main.js",
+  "main": "./dist/main.js",
+  "types": "./dist/main.d.ts",
+  "type": "module",
   "scripts": {
     "build": "tsc",
-    "format": "prettier --write '**/*.ts'",
+    "format": "prettier --write .",
     "lint": "eslint src/**/*.ts",
-    "package": "ncc build --source-map --license licenses.txt",
-    "all": "npm run build && npm run format && npm run lint && npm run package"
+    "package": "ncc build",
+    "all": "pnpm run format && pnpm run lint && pnpm run build && pnpm run package"
```

Comment:
run prettier too? or does lint check prettier? might be nice either way, if we check prettier on CI, to know locally first (not that I forget and have to push prettier fixup commits...)

Reply:
`format` does that :)